### DISCUSSION TITLE
Describe outputs

### DISCUSSION THREAD

**Discussion by spxyx** [lllyasviel/Fooocus#3342] [General]
I used "Describe" for a picture and it output this at the end of the prompt: \(medium\)
What does this control? Is there a list of controls for prompts? I looked in the docs and did not see it.

**Comment by ILikeToasters** [comment]
I am not sure why it would give that and without knowing the whole prompt I could not really guess but all the describe function does is fill in the prompt with a description of the picture. If it included (medium) all the parentheses do is add weight to the word when in a prompt. For example (cat) would have more weight in the prompt than just cat. And ((cat)) would have even more weight. Weight just means you are telling Fooocus you want that word to be more important. Weight can also be expressed such as (cat:1.1) which would be the same as (cat) and (cat:1.2) would be the same as ((cat)). I prefer using the actual numbers myself since it is easier to read. You can also use (cat:0.8) for example for less weight. 

You can easily adjust the weight using the CTRL+up/down arrows on a word so you don't have to type in the parentheses and numbers.

**Comment by spxyx** [comment]
I just realized that my first message dropped the forward slashes that proceed the open and close parenthesis around the work Medium.
